src/worker.py

```
@@ -7669,6 +7684,101 @@ async def _dispatch(request, env):
+                    r2_bucket = (
+                        getattr(env, "MY_BUCKET", None)
+                        or getattr(env, "R2_BUCKET", None)
+                        or getattr(env, "R2", None)
+                    )
+                if not r2_bucket:
+                    return err("R2 storage not configured", 500)
+                obj = await r2_bucket.get(r2_key)
+                if obj is None:
+                    return err("File not found", 404)
+                try:
+                    from pyodide.ffi import to_js  # noqa: PLC0415
```

Flake8 (7.3.0)

[error] 'pyodide.ffi.to_js' imported but unused

(F401)